wdl/TasksClusterBatch.wdl

```diff
@@ -0,0 +1,504 @@
+version 1.0
+
+import "Structs.wdl"
+
+task SVCluster {
+    input {
+        # Either vcfs of vcfs_tar should be provided
+        Array[File] vcfs = []  # Can't use optional because of write_lines() call
+        File? vcfs_tar
+
+        File ploidy_table
+        String output_prefix
+
+        String? contig
+
+        Boolean? fast_mode
+        Boolean? omit_members
+        Boolean? enable_cnv
+        Boolean? default_no_call
+
+        String? algorithm
+        String? insertion_length_summary_strategy
+        String? breakpoint_summary_strategy
+        String? alt_allele_summary_strategy
+
+        Float? defrag_padding_fraction
+        Float? defrag_sample_overlap
+
+        Float? depth_sample_overlap
+        Float? depth_interval_overlap
+        Int? depth_breakend_window
+        Float? mixed_sample_overlap
+        Float? mixed_interval_overlap
+        Int? mixed_breakend_window
+        Float? pesr_sample_overlap
+        Float? pesr_interval_overlap
+        Int? pesr_breakend_window
+
+        File reference_fasta
+        File reference_fasta_fai
+        File reference_dict
+
+        Float? java_mem_fraction
+        String? variant_prefix
+
+        String gatk_docker
+        RuntimeAttr? runtime_attr_override
+    }
+
+    parameter_meta {
+        vcfs: {
+                  localization_optional: true
+              }
+    }
+
+    RuntimeAttr default_attr = object {
+                                   cpu_cores: 1,
+                                   mem_gb: 3.75,
+                                   disk_gb: ceil(10 + size(vcfs, "GB") * 3 + size(reference_fasta, "GB")),
+                                   boot_disk_gb: 10,
+                                   preemptible_tries: 3,
+                                   max_retries: 1
+                               }
+    RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
+
+    output {
+        File out = "~{output_prefix}.vcf.gz"
+        File out_index = "~{output_prefix}.vcf.gz.tbi"
+    }
+    command <<<
+        set -euo pipefail
+
+        function getJavaMem() {
```

Comment:
This seems potentially like a generally useful function -- it might be neat to try to add it somehow to the gatk docker so that it's always there. If you agree you could make a ticket on the gatk repo for someone to do it.